cypress/TestCases/FireboltCertification/UserGrants.md

```diff
@@ -0,0 +1,56 @@
+# Firebolt Certification
+Feature file for certifying UserGrants module.
+
+## Purpose
+A module for managing grants given by the user.
+
+## Requirements :
+1.**Grant Policies**
```

Comment:
Under Requirements, give a brief introduction 

Usergrant test cases present in FCS are using the grantPolicies listed in grantPolicy.json (this should be a hyperlink which when clicked will go to the grantPolicy,json file) for validation. Each api used for validating usergrants has a grantPolicy defined in this file with different values for scope, lifespan, grantSteps etc. GrantPolicy structure is outlined in the spec document. 


In order for a user to execute Usergrant test cases, they would need additional implementation in config module .

Reply:
all the glue code explanation need not be here, we can just provide a link here